scripts/setup-auth-qa.sh

```diff
@@ -13,8 +13,15 @@ fi
 
 QA="${PROJECT_NAME}QA"
 
+read -p "Enter login domain: (test.salesforce.com) " LOGIN_DOMAIN
+if [ -z $LOGIN_DOMAIN ] ; then
+    LOGIN_DOMAIN='test.salesforce.com'
```

Comment:
Actually just tested and found the protocol is required (test.salesforce.com fails, it has to be https://test.salesforce.com)

Reply:
To test locally:

Fails = `sfdx force:auth:web:login -a test -r test.salesforce.com`
Succeeds = sfdx force:auth:web:login -a test -r https://test.salesforce.com`